packages/vscode-lit-xml/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "vscode-lit-xml",
+  "displayName": "lit-xml",
+  "description": "Syntax highlighting xml inside of JavaScript and TypeScript lit-xml tagged template strings",
+  "publisher": "hugo-vrijswijk",
+  "license": "Apache-2.0",
+  "version": "0.2.0",
+  "main": "dist/src/index",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nicojs/lit-xml.git"
+  },
+  "scripts": {
+    "publish": "vsce publish",
+    "build": "vsce package"
+  },
+  "engines": {
+    "vscode": "^1.32.0"
+  },
+  "categories": [
+    "Programming Languages"
+  ],
+  "keywords": [
+    "lit-xml",
+    "javascript",
+    "typescript",
+    "xml",
+    "template"
+  ],
+  "activationEvents": [
+    "onLanguage:javascript",
+    "onLanguage:javascriptreact",
+    "onLanguage:typescript",
+    "onLanguage:typescriptreact"
+  ],
+  "contributes": {
+    "grammars": [
+      {
+        "injectTo": [
+          "source.js",
+          "source.js.jsx",
+          "source.jsx",
+          "source.ts",
+          "source.tsx",
+          "text.xml"
+        ],
+        "scopeName": "text.lit-xml",
+        "path": "./syntaxes/lit-xml.json",
+        "embeddedLanguages": {
+          "meta.embedded.block.xml": "xml"
+        }
+      },
+      {
+        "injectTo": [
+          "source.js",
+          "source.js.jsx",
+          "source.jsx",
+          "source.ts",
+          "source.tsx",
+          "text.xml"
```

Comment:
Unsure about this one. Does it need to activate in XML documents?

Reply:
Good catch! It needs to be `text.html` to activate in html files
![image](https://user-images.githubusercontent.com/10114577/81533986-79a51d00-9367-11ea-8c06-f1caa0fbc8a7.png)